lib/hatchet/tasks.rb

```diff
@@ -1,7 +1,11 @@
 namespace :hatchet do
   task :setup_ci do
     script = File.expand_path(File.join(__dir__, "../../etc/ci_setup.rb"))
-    out = `#{script}`
+    puts "Running script `#{script}`"
+    out = `#{script} 2>&1`
+    puts "Done"
+    puts "Output:\n"
+    puts out
```

Comment:
I think the output will be shown twice if the command fails. Should some of these new lines be moved after the `raise`, so they are only output if the failure message hasn't been printed?

Reply:
I removed the duplicate output from the failure case, good catch.